plugins/MauticFocusBundle/Model/FocusModel.php

```diff
@@ -411,14 +411,14 @@ public function getStats(Focus $focus, $unit, \DateTime $dateFrom = null, \DateT
 
         if ($focus->getType() != 'notification') {
             if ($focus->getType() == 'link') {
-                $q = $query->prepareTimeDataQuery('focus_stats', 'date_added', ['type' => Stat::TYPE_CLICK]);
+                $q = $query->prepareTimeDataQuery('focus_stats', 'date_added', ['type' => Stat::TYPE_CLICK,'focus_id' => $focus->getId()]);
```

Comment:
Fix the code standards, please.

https://travis-ci.org/mautic/mautic/jobs/295299190#L1340

There should be a space after the comma.

Reply:
Have added the space.